src/components/Signup/Signup.js

```diff
@@ -2,26 +2,26 @@ import React, { Component, PropTypes } from 'react';
 import { Link } from 'react-router';
 
 import '../../styles/Signup.scss';
+import '../../styles/Common.scss';
 import SignupForm from './SignupForm';
 
 class Signup extends Component {
   render() {
     const { submitSignup, signupError } = this.props;
 
     return (
-      <div>
-        <div>
-          <h2>Signup</h2>
-          <SignupForm
-            submitSignup={submitSignup}
-          />
-          {
-            signupError ?
-            <span>{signupError}</span> :
-            null
-          }
-          <Link to="/">Log In</Link>
-        </div>
+      <div className="container">
+        <h1 className="title">SIGN UP</h1>
```

Comment:
Do not use uppercase text, instead add `text-transform: uppercase` on `title` class.

Same for all others.